frigate/util.py

```diff
@@ -1210,3 +1211,66 @@ def calculate_duration(video: Optional[any]) -> float:
         result["height"] = round(height)
 
     return result
+
+
+def update_yaml_from_url(file_path, url):
```

Comment:
Can we make this function more generic so that it accepts just a dictionary or list? This way we can use this function more easily for updating the config via mqtt and other parts of the code.

Something like:

```python
{
  "cameras.back.motion.mask": "0,0,10,10,..."
}
```

Reply:
1926c1d